content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -114,7 +114,8 @@ For example, to install the controller and a default version of Istio into your
 run the following command:
 
 {{< text bash >}}
-$ kubectl apply -f https://<repo URL>/<version>/operator-profile-default.yaml
+$ kubectl apply -f https://<repo URL>/operator.yaml
+$ kubectl apply -f https://<repo URL>/<version>/default-cr.yaml
```

Comment:
@ostromart I'm not sure I understand what exactly this <repo URL> and <version> in the path are supposed to be. What repo should this be? Why is the version a path component like this?

Reply:
The URL is TBD which is why I left it like that - we also haven't started creating subdirs for releases. This is just my best guess how this would work. I think it may be better to just leave the upgrade section out, since this is still somewhat in flux.

Reply:
@ostromart the queestion i hae is what is `<repo URL>` - Is that a repository we make specifically for deploying the operator?  If so, the grab script could go there, and with a little maekfile magic, we could move this work from the documentation repo - which was probably the wrong place for it originally.